kotlin.html.markdown

```diff
@@ -2,14 +2,15 @@
 language: kotlin
 contributors:
     - ["S Webber", "https://github.com/s-webber"]
+    - ["Jakukyo Friel", "https://weakish.github.io"]
```

Comment:
While we do appreciate your contribution, but please do not add yourself to contributors for only formatting changes.